nucliadb_vectors/src/relations/bfs_engine.rs

```diff
@@ -18,15 +18,14 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 //
 
-use std::collections::{HashSet, LinkedList};
+use std::collections::{HashMap, HashSet, LinkedList};
 
 use super::errors::*;
 use crate::relations::graph_db::*;
 
 // BfsGuide allows the user to modify how the search will be performed.
 // By default a BfsGuide does not interfere in the search.
 pub trait BfsGuide {
-    fn matches(&self, node: Entity) -> bool;
```

Comment:
What is the rational behind removing this method ? Is it some dead code ?

Reply:
I don't know, @hermeGarcia delete it